Sources/BraintreeVenmo/BTVenmoRequest.swift

```diff
@@ -22,78 +22,73 @@ import Foundation
-    /// Also, `paymentMethodUsage` on the BTVenmoRequest must be set to `.multiUse`.
-    ///
-    /// If this property is set to `false`, you can still vault the Venmo account on your server, provided that `paymentMethodUsage` is not set to `.singleUse`.
-    /// Defaults to `false`
-    public var vault: Bool = false
-
-    /// If set to `.multiUse`, the Venmo payment will be authorized for future payments and can be vaulted.
-    ///  If set to `.singleUse`, the Venmo payment will be authorized for a one-time payment and cannot be vaulted.
-    public var paymentMethodUsage: BTVenmoPaymentMethodUsage
-
-    /// Optional. The business name that will be displayed in the Venmo app payment approval screen. Only used by merchants onboarded as PayFast channel partners.
-    public var displayName: String?
+    // MARK: - Internal Properties
     
-    /// Whether the customer's billing address should be collected and displayed on the Venmo paysheet.
-    /// Defaults to `false`
-    public var collectCustomerBillingAddress: Bool = false
-    
-    /// Whether the customer's shipping address should be collected and displayed on the Venmo paysheet.
-    /// Defaults to `false`
-    public var collectCustomerShippingAddress: Bool = false
-
-    /// Indicates whether the purchase amount is the final amount.
-    /// Removes "subject to change" notice in Venmo app paysheet UI.
-    /// Defaults to `false`
-    public var isFinalAmount: Bool = false
-
-    /// Optional. The subtotal amount of the transaction to be displayed on the paysheet. Excludes taxes, discounts, and shipping amounts.
-    ///
-    /// If this value is set, `totalAmount` must also be set.
-    public var subTotalAmount: String?
-    
-    /// Optional. The total discount amount applied on the transaction to be displayed on the paysheet.
-    ///
-    /// If this value is set, `totalAmount` must also be set.
-    public var discountAmount: String?
-    
-    /// Optional. The total tax amount for the transaction to be displayed on the paysheet.
-    ///
-    /// If this value is set, `totalAmount` must also be set.
-    public var taxAmount: String?
-    
-    /// Optional. The shipping amount for the transaction to be displayed on the paysheet.
-    ///
-    /// If this value is set, `totalAmount` must also be set.
-    public var shippingAmount: String?
-    
-    /// Optional. The grand total amount on the transaction that should be displayed on the paysheet.
-    public var totalAmount: String?
-    
-    /// Optional. The line items for this transaction. It can include up to 249 line items.
-    ///
-    /// If this value is set, `totalAmount` must also be set.
-    public var lineItems: [BTVenmoLineItem]?
-
-    /// Optional. Used to determine if the customer should fallback to the web flow if Venmo app is not installed.
-    ///
-    /// Defaults to `false`
-    public var fallbackToWeb: Bool = false
+    var paymentMethodUsage: BTVenmoPaymentMethodUsage
```

Comment:
I was thinking, do these properties need to be `var`, or can we change them to `let`?

Reply:
https://github.com/braintree/braintree_ios/pull/1439#discussion_r1811083484